src/lib/form-field/_form-field-theme.scss

```diff
@@ -113,11 +113,19 @@
 // technically different but still render the same by adding a tiny value to the transform / width.
 @mixin _mat-form-field-placeholder-float-nodedupe($font-scale, $infix-padding, $infix-margin-top) {
   transform: translateY(-$infix-margin-top - $infix-padding) scale($font-scale) perspective(100px)
-             translateZ(0.002px);
-  -ms-transform: translateY(-$infix-margin-top - $infix-padding) scale($font-scale);
+             translateZ(0.001px) scale(1);
+  -ms-transform: translateY(-$infix-margin-top - $infix-padding) scale($font-scale) scale(1);
   width: (100% / $font-scale) + 0.0001;
 }
 
+@mixin _mat-form-field-placeholder-float-nodedupe2($font-scale, $infix-padding, $infix-margin-top) {
```

Comment:
Instead of adding another mixin, I realized we could programatically change the rule each time it's stamped out, e.g.:
```scss
$uniqueId: 0;

@mixin cool-style() {
  content: 'Hello' + $uniqueId;
  $uniqueId: $uniqueId + 1 !global;
}


.one {
  @include cool-style(); // content: "Hello0";
}

.two {
  @include cool-style(); // content: "Hello1";
}
```